python/ray/autoscaler/autoscaler.py

```diff
@@ -342,18 +347,18 @@ def __init__(self,
         for local_path in self.config["file_mounts"].values():
             assert os.path.exists(local_path)
 
-        print("StandardAutoscaler: {}".format(self.config))
+        log.info("StandardAutoscaler: {}".format(self.config))
```

Comment:
See, this is an example where if we would not pre-format the message before passing it to `log.info`, whole config would be stored into the logging record and one could extract it out directly, instead of serialized to the string.

Reply:
I'm fine with this. Extracting the config would be a niche use case and you can always do that with a regex.